src/main/java/pl/betoncraft/betonquest/utils/Utils.java

```diff
@@ -252,70 +252,70 @@ public static void loadDatabaseFromBackup() {
                     con.updateSQL(UpdateType.INSERT_POINT,
-                            new String[]{points.getString(key + ".id"), points.getString(key + ".playerID"),
-                                    points.getString(key + ".category"), points.getString(key + ".count"),});
+                            points.getString(key + ".id"), points.getString(key + ".playerID"),
+                            points.getString(key + ".category"), points.getString(key + ".count"));
                 }
             }
             // load journals
             final ConfigurationSection journals = config.getConfigurationSection("journals");
             if (journals != null) {
                 for (final String key : journals.getKeys(false)) {
                     con.updateSQL(UpdateType.INSERT_JOURNAL,
-                            new String[]{journals.getString(key + ".id"), journals.getString(key + ".playerID"),
-                                    journals.getString(key + ".pointer"), journals.getString(key + ".date"),});
+                            journals.getString(key + ".id"), journals.getString(key + ".playerID"),
+                            journals.getString(key + ".pointer"), journals.getString(key + ".date"));
                 }
             }
             // load backpack
             final ConfigurationSection backpack = config.getConfigurationSection("backpack");
             if (backpack != null) {
                 for (final String key : backpack.getKeys(false)) {
                     con.updateSQL(UpdateType.INSERT_BACKPACK,
-                            new String[]{backpack.getString(key + ".id"), backpack.getString(key + ".playerID"),
-                                    backpack.getString(key + ".instruction"), backpack.getString(key + ".amount"),});
+                            backpack.getString(key + ".id"), backpack.getString(key + ".playerID"),
+                            backpack.getString(key + ".instruction"), backpack.getString(key + ".amount"));
                 }
             }
             // load player
             final ConfigurationSection player = config.getConfigurationSection("player");
             if (player != null) {
                 for (final String key : player.getKeys(false)) {
                     con.updateSQL(UpdateType.INSERT_PLAYER,
-                            new String[]{player.getString(key + ".id"), player.getString(key + ".playerID"),
-                                    player.getString(key + ".language"), player.getString(key + ".conversation")});
+                            player.getString(key + ".id"), player.getString(key + ".playerID"),
+                            player.getString(key + ".language"), player.getString(key + ".conversation"));
                 }
             }
             // delete backup file so it doesn't get loaded again
             file.delete();
-        } catch (Exception e) {
+        } catch (SecurityException e) {
```

Comment:
Catching anything here isn't useful, as the `updateSQL` calls all catch SqlExceptions and don't tell you if they fail. Again only `NullPointerException`s possible as there are some missing `null` checks with the `java.io.File` code. Just remove the try-catch.